client/src/app/controllers/socket/index.coffee

```diff
@@ -0,0 +1,14 @@
+activity = require("../activity/activity.collection")
+
+class Socket
+  constructor: -> @initialize()
+
+  initialize: ->
+    # Connect to socket server
+    @socket = io("http://localhost:3000")
```

Comment:
This can't be hardcoded.  It could be defaulted to `localhost:3000` but needs to support being set a to a "production" endpoint or alternate endpoint